lib/logger/package.json

```diff
@@ -27,7 +27,7 @@
     "ansi-colors": "^4.1.1",
     "ansi-regex": "^5.0.1",
     "triple-beam": "^1.3.0",
-    "winston": "^3.5.0",
+    "winston": "^3.4.0",
```

Comment:
**suggestion**: feels like this ought to be `^3.5.1` so there's no chance of it getting the broken `3.5.0`?